erts/emulator/beam/beam_load.c

```diff
@@ -164,7 +167,7 @@ erts_prepare_loading(Binary* magic, Process *c_p, Eterm group_leader,
         BeamLoadError2(stp,
                        "This BEAM file was compiled for a later version"
                        " of the run-time system than " ERLANG_OTP_RELEASE ".\n"
-                       "  To fix this, please recompile this module with an "
+                       "  To fix this, please recompile this module with an OTP "
```

Comment:
```suggestion
                       "  To fix this, please recompile this module with an Erlang/OTP "
```

I think it is better to say Erlang/OTP to be clearer.